frontend/src/Editor/SubContainer.jsx

```diff
@@ -70,6 +70,10 @@ export const SubContainer = ({
     shallow
   );
 
+  const { selectedComponents } = useEditorStore((state) => ({
+    selectedComponents: state.selectedComponents,
+  }));
+
```

Comment:
Do we need this? It's used on `onDragStop` function and we can use the `getState` here

Reply:
Resolved. Used getState here.